knowledge-base/error-resource-not-found-after-finishing-the-item-template-wizard.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Error 404 Resource not found
+description: After finishing the item template wizard for creating HTML5-based report viewer with either separate hosing of the Reporting REST Service or Report Server connection, an error 404 is thrown in the browser console.
+type: troubleshooting
+page_title: Error 404 Resource not found after creating a HTML-based Report Viewer using the item template wizard
+slug: error-resource-not-found-after-finishing-the-item-template-wizard
+position: 
+tags: HTML5Viewer, MVCReportViewer, RESTService, WebFormsViewer, AngularViewer
+ticketid: 1428873
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product Version</td>
+			<td>13.1.19.514</td>
+		</tr>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+		<tr>
+			<td>Project Type</td>
+			<td>Web application</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+This article elaborates on an error for not found resource after creating a new HTML5-based report viewer using [the built-in item template wizard](https://docs.telerik.com/reporting/html5-report-viewer-quick-start). 
+When it comes to hosting the Report Viewer and Reporting REST Service (or Report Server) on different domains, the generated script reference has a relative path trying to find the resource within the viewer application host. Instead, it should try to get the resource from the REST Service host.
+
+## Steps to Reproduce
+The error might occur in two cases:
+1. In case there is a REST service already configured on a different host. On 'Configure reporting engine' step of the item template wizard, choosing Rest service option and "Use existing REST service" where you enter a valid REST service URI.
```

Comment:
If I were you, I would say In <bold>Configure reporting engine</bold> dialog, choosing ....

Reply:
fixed